frontend/packages/dev-console/src/components/import/pipeline/pipeline-template-utils.ts

```diff
@@ -40,13 +40,14 @@ export const createPipelineForImportFlow = async (formData: GitImportFormData) =
     labels: template.metadata.labels,
   };
 
-  template.spec.params = template.spec.params.map((param) => {
-    if (param.name === 'APP_NAME') {
-      param.default = name;
-    }
-
-    return param;
-  });
+  template.spec.params =
+    template.spec.params &&
+    template.spec.params.map((param) => {
+      if (param.name === 'APP_NAME') {
+        param.default = name;
+      }
+      return param;
+    });
```

Comment:
If there are no params, this will result in `template.spec.params = false` which I don't think is what you want to do. Need this surrounded by an if condition.